### PR TITLE
updates eslint based Vite

### PR DESCRIPTION
**Pull request by fpigeonjr**
updates eslint based Vite
- fixes linter issues

## What changed

updates eslint based Vite

## Issue

#1111 

## How to test

run `yarn lint` 

## Links

_If relevant, e.g. for a link to a piece of markdown documentation_

### CODE REVIEW

File: frontend/src/api/patchAgreements.js
Comment (fpigeonjr) [inline frontend/src/api/patchAgreements.js:1]: @stevtek, may need some help resolving these linter errors here.

**Review by stevtek — APPROVED** [review]

File: frontend/package.json [hunk 1]
```diff
@@ -44,10 +44,13 @@
         "cypress": "12.17.1",
         "cypress-axe": "1.4.0",
         "cypress-localstorage-commands": "2.2.3",
-        "eslint-config-prettier": "8.8.0",
+        "eslint-config-prettier": "^8.8.0",
```
Comment (tdonaworth) [inline frontend/package.json:47]: don't we want to pin to a specific version?

File: frontend/package.json [hunk 2]
```diff
@@ -44,10 +44,13 @@
...
         "eslint-plugin-cypress": "2.13.3",
         "eslint-plugin-import": "2.27.5",
         "eslint-plugin-prettier": "4.2.1",
+        "eslint-plugin-react": "^7.32.2",
```
Comment (tdonaworth) [inline frontend/package.json:51]: same, we should pin to the exact version on these.

**Review by tdonaworth — CHANGES_REQUESTED** [review]
version pinning

**Review by johndeange — APPROVED** [review]

**Review by tdonaworth — APPROVED** [review]

**Pull request merged**